bin/pulsar

```diff
@@ -204,15 +204,15 @@ elif [ $COMMAND == "zookeeper" ]; then
     PULSAR_LOG_FILE=${PULSAR_LOG_FILE:-"zookeeper.log"}
 
     # Add instrumentation
-    WEAVER_JAR=`find $PULSAR_HOME/lib -name 'aspectjweaver-*.jar'`
+    WEAVER_JAR=`find $PULSAR_HOME/lib -name 'org.aspectj-aspectjweaver-*.jar'`
```

Comment:
a small suggestion: can't we load it dynamically in `ZooKeeperStarter` similar way we are doing at [PulsarBrokerStarter](https://github.com/apache/incubator-pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java#L60)

Reply:
Good point. That's much easier.